Makefile

```diff
@@ -21,7 +22,7 @@ clean:
 
 docker:
 	@echo ">> Compile using docker container"
-	@docker build -t "$(DOCKER_IMAGE_NAME)" .
+	@docker build --build-arg VERSION=$(VERSION) -t $(DOCKER_IMAGE_NAME):latest .
```

Comment:
Should't we use the version tag instead of latest?

Reply:
This is how build args are used
https://docs.docker.com/build/guide/build-args/

Are we using VERSION arg inside the Dockerfile? if not, then adding this build arg won't be of any use.
We should tag the image as "latest" as well as with the "<version>"

Reply:
Added VERSION arg inside dockerfile